packages/cozy-client/src/store/documents.js

```diff
@@ -131,6 +131,29 @@ export const getDocumentFromSlice = (state = {}, doctype, id) => {
   return state[doctype][id]
 }
 
+export const getDocumentsFromSlice = (state = {}, doctype, ids) => {
+  if (!doctype) {
+    throw new Error(
+      'getDocumentFromSlice: Cannot retrieve documents with undefined doctype'
+    )
+  }
+  if (!ids || ids.length < 1) {
+    throw new Error(
+      'getDocumentFromSlice: Cannot retrieve documents with undefined ids'
+    )
+  }
+  if (!state[doctype]) {
+    if (process.env.NODE_ENV !== 'production') {
+      logger.info(
+        `getDocumentFromSlice: ${doctype} is absent from the store's documents. State is`,
+        state
+      )
+    }
+    return null
```

Comment:
I would suggest returning an empty array to always have the same return type.

Reply:
That is correct, as a punishement, I added the jsdoc to both `getDocumentFromSlice` and `getDocumentsFromSlice`